sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -1918,6 +1918,27 @@ object GpuOverrides {
         override def convertToGpu(): GpuExpression =
           GpuCreateNamedStruct(childExprs.map(_.convertToGpu()))
       }),
+    expr[ArrayContains](
+      "Returns a boolean if the array contains the passed in key",
+      ExprChecks.binaryProjectNotLambda(
+        TypeSig.BOOLEAN,
+        TypeSig.all,
+        ("array", TypeSig.ARRAY.nested(TypeSig.commonCudfTypes + TypeSig.NULL),
```

Comment:
Why are we not testing for decimal support.  This is a new operator and there is no reason we should not start off with decimal support.

Reply:
I don't think decimal is supported for array contains on cudf side.